roaring.go

```diff
@@ -1713,3 +1738,15 @@ func (rb *Bitmap) Stats() Statistics {
 	}
 	return stats
 }
+
+// getNewArrayContainer checks the arrayContainerPool for an existing array
+// container, and if it doesn't find one it just allocates a new one.
+func (rb *Bitmap) getNewArrayContainer() container {
+	if len(rb.arrayContainerPool) > 0 {
+		newac := rb.arrayContainerPool[len(rb.arrayContainerPool)-1]
+		rb.arrayContainerPool = rb.arrayContainerPool[:len(rb.arrayContainerPool)-1]
```

Comment:
I think you you should set the last entry in the slice to `nil` after you initialize `newac` so it is possible to GC it sooner

Reply:
good idea